train.py

```diff
@@ -408,11 +427,12 @@ def main(args, verbose=True, optuna_trial=None, cmd_args=None):
         model_args = {
             "encoder_name": "resnet18",
             "classes": 1,
+            "in_channels": 1,
             "activation": "sigmoid",
             "encoder_weights": None,
         }
         model = smp.Unet(**model_args)
-        model.encoder.conv1 = nn.Sequential(nn.Conv2d(1, 3, 1), model.encoder.conv1)
+        # model.encoder.conv1 = nn.Sequential(nn.Conv2d(1, 3, 1), model.encoder.conv1)
```

Comment:
Da meintest du, dass du das über in_channels im constructor machst richtig @a1302z ? +1

Reply:
genau, ist in zeile 430